modules/openapi-generator/src/main/resources/C-libcurl/CMakeLists.txt.mustache

```diff
@@ -80,20 +81,73 @@ set(HDRS
 
 )
 
+set(VERSION 0.0.1)
```

Comment:
Could you please move `set(VERSION 0.0.1)` near after `set(pkgName "{{projectName}}")` (line 30) ? 
I think it's clear for user.

Reply:
And add an annotation to tell user the "VERSION" can be changed by some other file.

Reply:
Done